src/components/DocumentationTopic/PrimaryContent/Mentions.vue

```diff
@@ -0,0 +1,56 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2024 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <section v-if="topics.length > 0" class="mentions">
```

Comment:
Minor thing, but you can shorten this to `topics.length` because anything over 0 is considered "truthy" in JavaScript (and 0 is "falsy").

It may also be better to move this conditional to the thing rendering this component instead of the component itself.